app/views/shared/_flash.html.erb

```diff
@@ -1,5 +1,6 @@
 <% flash.each do |key, message| %>
-  <div class="<%= flash_class(key) %>" role="alert">
-    <%= message %>
+  <div class="<%= flash_class(key) %> alert-dismissible" role="alert">
+    <a href="#" class="close" data-dismiss="alert" aria-label="close" style="text-decoration: none;"><%= fa_icon('times') %></a>
+    <%= value %>
```

Comment:
Obrigado! :bowing_man:

Reply:
Ah! One tiny fix (this should get Travis passing):
```suggestion
    <%= message %>
```

Reply:
De nada! Não precisa agradecer. ✋